spec/services/customers/create_service_spec.rb

```diff
@@ -62,6 +63,29 @@
     expect(SendWebhookJob).to have_received(:perform_later).with("customer.created", result.customer)
   end
 
+  context "when organization has multiple billing entities" do
```

Comment:
this scenario shouldn't happen, but we have this error raise, what do you think about adding a scenario when an organization has no active billing_entities
or better:
a code for discarded or archived billing_entity is provided

Reply:
done added for create, update and upsert from api